website/src/pages/admin/messages/[id].tsx

```diff
@@ -17,7 +17,7 @@ const MessageDetail = () => {
   const { data, isLoading, error } = useSWRImmutable<{
     tree: MessageWithChildren | null;
     message?: Message;
-  }>(`/api/admin/messages/${messageId}/tree`, get);
+  }>("/api/admin/messages", () => get(`/api/admin/messages/${messageId}/tree`));
```

Comment:
Revert this too

Reply:
I think this is correct.  You are modifying /api/admin/messages

Reply:
Done